### DISCUSSION TITLE
Ajout du service Eskimi

### DISCUSSION THREAD

**Discussion by TVanmeerhaeghe** [AmauriC/tarteaucitron.js#1119] [Q&A]
Bonjour,

Je souhaiterais ajouter un service pour Eskimi.
J'ai ajouter ceci dans tarteaucitron.services.js : 
`
tarteaucitron.services.eskimi = {
  key: "eskimi",
  type: "ads|analytic",
  name: "eskimi",
  needConsent: true,
  cookies: ["eskimi"],
  readmoreLink: "/#tarteaucitron",
  js: function () {
    "use strict";
    !(function (f, e, t, u, n, s, p) {
      if (f.esk) return;
      n = f.esk = function () {
        n.callMethod
          ? n.callMethod.apply(n, arguments)
          : n.queue.push(arguments);
      };
      if (!f.___esk) f.___esk = n;
      n.push = n;
      n.loaded = !0;
      n.queue = [];
      s = e.createElement(t);
      s.async = !0;
      s.src = u;
      p = e.getElementsByTagName(t)[0];
      p.parentNode.insertBefore(s, p);
    })(
      window,
      document,
      "script",
      "https://dsp-media.eskimi.com/assets/js/e/gtr.min.js?_=0.0.0.4"
    );
    esk("init", "31261");
    tarteaucitron.addScript(
      "https://dsp-media.eskimi.com/assets/js/e/gtr.min.js?_=0.0.0.4"
    );
  },
  fallback: function () {
    "use strict";
  },
};
`

Puis ajouter ceci dans ma balise head : 
`
(tarteaucitron.job = tarteaucitron.job || []).push("eskimi");
`

Mais je n'arrive pas à faire en sorte que mon script ne se déclenche que si l'utilisateur accepte mes cookies.

Je vous remercie pour votre aide !

**Comment by AmauriC** [comment]
Le `type` ne doit pas comporter de |, il faut choisir entre ads et analytics.

Il faut également retirer du site le marqueur html Eskimi et laisser tarteaucitron gérer le chargement.

La ligne `tarteaucitron.addScript( "https://dsp-media.eskimi.com/assets/js/e/gtr.min.js?_=0.0.0.4" );` fait doublon avec le reste du code.

Possible d'avoir le marqueur html de base complet pour que je fasse un service conforme et que je l'ajoute à la liste? Merci ; )

**Comment by TVanmeerhaeghe** [comment]
Merci pour votre réponse rapide.
Voici ce qu'on m'as fournis : 
`<script>
    !function(f,e,t,u,n,s,p) {if(f.esk)return;n=f.esk=function(){n.callMethod?n.callMethod.apply(n,arguments):n.queue.push(arguments)};if(!f.___esk)f.___esk=n;n.push=n;n.loaded=!0;n.queue=[];s=e.createElement(t);s.async=!0;s.src=u;p=e.getElementsByTagName(t)[0];p.parentNode.insertBefore(s,p)}(window,document,'script', '[https://dsp-media.eskimi.com/assets/js/e/gtr.min.js?_=0.0.0.4'](https://dsp-media.eskimi.com/assets/js/e/gtr.min.js?_=0.0.0.4%27));
    esk('init', '31261');
</script>`

J'espère que c'est bien cela que vous entendez par "marqueur html"
Merci pour l'ajout !

**Reply by AmauriC** [reply]
Parfait, je m'en occupe rapidement (https://github.com/AmauriC/tarteaucitron.js/issues/1120).
